testing/btest/Baseline/coverage.bare-mode-errors/errors

```diff
@@ -1,15 +1,2 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
 ### NOTE: This file has been sorted with diff-sort.
-warning in <...>/__load__.zeek, line 1: deprecated script loaded from <...>/__load__.zeek:2 "Remove in v7.1 The policy<...>/defaults package is deprecated. The options set here are now the defaults for Zeek in general.";
```

Comment:
The corresponding test has a few `grep -v` lines for other deprecated scripts. Same for `test-all-policy-cluster`.

I've added them, but I start to like the approach of including the warnings, too, will do next time!